packages/amplify-graphql-maps-to-transformer/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "devDependencies": {
     "@aws-amplify/graphql-model-transformer": "0.16.0",
+    "@aws-amplify/graphql-index-transformer": "0.14.0",
```

Comment:
there's index transformer usage in the `with-many-to-many.test.ts` . Magically latest version from build was picked up so adding this here for 1. completeness 2. to not bloat this PR with one more transformer.